README.md

```diff
@@ -35,6 +35,10 @@ Obtain the publishable and secret keys from the developer test mode on the Strip
 
 * Open the cloud-config.yaml file and replace the hard-coded name with your own.
 
+* Change the path on line 75 in vm_deploy.sh to /home/yourname
```

Comment:
You can just make the script more functional and use the $HOME variable in your command.

Reply:
Replaced /home/yourname with /home/$USER.